pos_category_multi/README.rst

```diff
@@ -0,0 +1,27 @@
+==================
+POS Category Multi
```

Comment:
You need spaces here. Check latest note: https://odoo-development.readthedocs.org/en/latest/dev/docs/index.html